uast/uast.go

```diff
@@ -410,6 +410,24 @@ const (
 
 	// Visibility is an access granter role, usually together with an specifier role
 	Visibility
+
+	// Annotation is syntactic metadata
+	Annotation
+
+	// Anonymous is an unbound construct
+	Anonymous
+
+	// Enumeration is a distinct type that represents a set of named constants
+	Enumeration
+
+	// Arithmetic is a type of operation
+	Arithmetic
+
+	// Relational is a type of operation
+	Relational
+
+	// Variable is a symbolic name associatend with a value
+	Variable
```

Comment:
This would be the strict definition of variables? (e. g. only with values that can change) or for the more relaxed usage of any symbol associated with data  (e. g. including immutable/const values)? If its the first case, we could also add a "Constant" role.

Reply:
The latter (matching the docstring). I agree a constant role may make sense, I have only added those I've used to add annotations in the java driver so far, which is not an exhaustive set, so it will probably be extended shortly